content/guides/why-webpack.md

```diff
@@ -4,29 +4,31 @@ contributors:
   - pksjce
   - bebraw
   - chrisVillanueva
+  - tashian
 sort: 2
 ---
 
-webpack is usually compared to tools like Make, Grunt, Gulp, Browserify or Brunch. However, some of these tools (Make, Grunt, and Gulp, which are task runners) have a much different purpose than webpack, which is a module bundler. Comparing them directly could lead to sorts of confusion, so let’s first draw a distinction between these types of tools.
+webpack is a module bundler, like Browserify or Brunch. It is not a task runner. Make, Grunt, or Gulp are task runners. But people get confused about the difference, so let's clear that up right away.
 
 ## What Are Task Runners?
 
-Task runners literally make it easier to handle tasks, such as linting, building, or developing your project. Compared to *bundlers* like Browserify, Brunch, or webpack, they have a higher level focus. Bundlers, on the other hand, have a much more specific goal.
-
+Task runners handle automation of common development tasks such as linting, building, or testing your project. Compared to *bundlers* like webpack, Browserify, or Brunch, task runners have a higher level focus. Bundlers, on the other hand, have a much more specific goal.
 
 ## What Are Bundlers?
 
-Roughly put bundlers take assets, such as JavaScript files in, and then transform them into format that's suitable for the browser of the end user to consume. This process of bundling happens to be one of the most important problems in web development and solving it well can remove a large part of pain from the process.
+Bundlers take assets, such as JavaScript files and modules, and transform them into a format that's suitable for the browser of the end-user to consume. As your project grows, this task becomes more complex. Maybe you want to split your code into multiple chunks that can be used on-demand. You want to minify your JavaScript and CSS when you deploy your application. You want your application to leverage caching well.
```

Comment:
> of the end-user 

Attention span for readers is a valuable real-estate. You can remove this snippet